### PR TITLE
chore: prep component minting

### PR DESCRIPTION
**Pull request by DavidMinarsch**

### CODE REVIEW

File: mints/mapping.txt [hunk 1]
```diff
@@ -1,8 +1,14 @@
 token_id-"component_id"
-X-
+X-"contract/valory/mech/0.1.0"
+X-"contract/valory/erc20/0.1.0"
+X-"contract/valory/market_maker/0.1.0"
```
Comment (jmoreira-valory) [inline mints/mapping.txt:4]: This name could collide with some components in the market-creator repository (e.g., agent/valory/market_maker/0.1.0, service/valory/market_maker:0.1.0)

**Review by jmoreira-valory — APPROVED** [review]

File: mints/mapping.txt (hunk 1 above)
Comment (DavidMinarsch) [reply mints/mapping.txt:4]: Are they the same or different contracts?

File: mints/04.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/market_maker:0.1.0","description":"Fixed product market maker contract","code_uri":"ipfs://bafybeighn4o263yxjxc46n5rckfnonwtyyhvjmegmkrqraixvswuyhcsp4","image":"ipfs://bafybeievzsyvjc3oyuwrnlceionzkdbhydoripjeuvlngbkhahjix2c7vq","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/04.json:1]: Why "fixed"?

File: mints/06.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"name":"skill/valory/decision_maker_abci:0.1.0","description":"ABCI application for common apps.","code_uri":"ipfs://bafybeigircx36tx3nw3r6dgym357e3wyzabb7y7zhahhttwbkgnxinsgre","image":"ipfs://bafybeiasruosr2t2qzv4iqoaj3l5idztygk5q36khovdfrdwzbddwmi3km","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```
Comment (DavidMinarsch) [inline mints/06.json:1]: This description of the skill is not a good one

File: mints/04.json (hunk 2 above)
Comment (Adamantios) [reply mints/04.json:1]: This is how they are called:

https://github.com/protofire/omen-subgraph#fixedproductmarketmaker

File: mints/06.json (hunk 3 above)
Comment (Adamantios) [reply mints/06.json:1]: Improved on https://github.com/valory-xyz/trader/pull/63.

**Pull request merged**